examples/step-59/doc/results.dox

```diff
@@ -0,0 +1,341 @@
+<h1>Results</h1>
+
+<h3>Program output</h3>
+
+Like in step-37, we evaluate the multigrid solver in terms of run time.  In
+two space dimensions with elements of degree 8, a possible output could look
+as follows:
+@code
+Running with 12 MPI processes, element FE_DGQHermite<2>(8)
+
+Cycle 0
+Number of degrees of freedom: 5184
+Total setup time              0.0282445 s
+Time solve (14 iterations)    0.0110712 s
+Verification via L2 error:    1.66232e-07
+
+Cycle 1
+Number of degrees of freedom: 20736
+Total setup time              0.0126282 s
```

Comment:
This is usually because the system has to load in the dynamic library code at the first time it hits a part of the code. Maybe also because not all processors have reached the full frequency yet after a very short time (I'm not sure how long time the program spends before main to know whether the time is so small that this could play in).